Components/Core/Spells/Equivalencies.lua

```diff
@@ -399,7 +399,7 @@ TMW.BE = {
 			 102543, -- Incarnation: King of the Jungle
 			 102560, -- Incarnation: Chosen of Elune
 			 106951, -- Berserk
-			-107574, -- Avatar
+			 107574, -- Avatar
```

Comment:
Don't need the `-` because there is only 1 version of Avatar for warriors.

Reply:
There are 15 different spells in the game named "Avatar" and all of them increase damage done. Checking by name here was intentional.